src/client/assets/css/Profile.css

```diff
@@ -10,11 +10,9 @@
 
 .top-container {
 	width: 100%;
-	border-width: 0.5px;
```

Comment:
- no, these lines of code make the styling ugly so it's unnecessary and should be removed